rest_api/operator_apis/network-operator-openshift-io-v1.adoc

```diff
@@ -582,7 +609,7 @@ Type::
 
 | `iptablesSyncPeriod`
 | `string`
-| The period that iptables rules are refreshed. Default: 30s
+| An internal kube-proxy parameter. In older releases of OCP, this sometimes needed to be adjusted in large clusters for performance reasons, but this is no longer necessary, and there is no reason to change this from the default value. Default: 30s
```

Comment:
Can we ask for `s/OCP/OpenShift Container Platform`?

Reply:
@bobfuru, we can ask for anything we want, I'm sure. :)

While the CNO is different, most API text lives in `openshift/api` repo and can be freely modified, although it requires rebuilding stuff with go and I've never gotten it to work. So in theory we can create PRs for many of the changes we might want.